manifest_indexing/manifest_indexing.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.6
+
+RUN git clone --single-branch --branch  feat/indexd_utils https://github.com/uc-cdis/gen3sdk-python ./gen3 && cd ./gen3 && git checkout 81d704d277779c68dbc45652f7273bac4ceec323 && cd ..
```

Comment:
Let's merge this after the `gen3sdk-python` PR merge: https://github.com/uc-cdis/gen3sdk-python/pull/19